README.md

```diff
@@ -21,10 +38,25 @@ Heya. I heard you like digging deep into Undertale data so I made a tool just fo
 * All core IO functionality extracted into a library for use in external tools.
 * Can generate an .yydebug file for the GM:S debugger so that you can edit variables live! (see [here](https://github.com/krzys-h/UndertaleModTool/wiki/Corrections-to-GameMaker-Studio-1.4-data.win-format-and-VM-bytecode,-.yydebug-format-and-debugger-instructions#yydebug-file-format))
 
-### Included scripts
+# Screenshots
+
+Here's some screenshots of what UTMT can do:
+
+## [AM2R: Horde Mode Mod](https://github.com/Hornoads/AM2R-The-Horde-Multitroid/releases)
+<img src="images/am2r.png" alt="AM2R: Horde Mode" width="640" height="480"/>
+
+## [Undertale: Bits and Pieces Mod](https://gamejolt.com/games/UndertaleBnP/574044)
+<img src="images/bnp-ut.png" alt="BnP" width="640" height="480"/>
+
+## [RIBBIT - The Deltarune Mod](https://gamejolt.com/games/ribbitmod/671888)
+<img src="images/ribbit-dr.png" alt="RIBBIT" width="640" height="480"/>
+
+# Included Scripts
+
 Included are some test scripts. They are, but not limited to:
+
 * Universal:
```

Comment:
This really should get renamed, or the scripts here should get reordered. EnableDebug, DebugToggler, GoToRoom and ShowRoomName for example are not universal.